aea/configurations/schemas/definitions.json

```diff
@@ -34,6 +35,54 @@
     "resource_name": {
       "type": "string",
       "pattern": "[a-zA-Z_][a-zA-Z0-9_]*"
+    },
+    "private_key_path": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "ledger",
+        "path"
+      ],
+      "properties": {
+        "ledger": {
+          "type": "string"
+        },
+        "path": {
+          "type": "string"
+        }
+      }
+    },
+    "ledger_api": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "ledger",
+        "addr",
+        "port"
+      ],
+      "properties": {
+        "ledger": {
+          "type": "string"
+        },
+        "addr": {
+          "type": "string"
+        },
+        "port": {
+          "type": "number"
+        }
+      }
+    },
+    "author": {
+      "type": "string",
+      "pattern": "[a-zA-Z0-9_]*"
+    },
+    "version": {
+      "type": "string",
+      "pattern": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
```

Comment:
We use semantic versioning for version strings (not backward compatible):
[https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string)

Reply:
Sounds good